### ISSUE TITLE
Using dprint toml fails with 0.5.4 wasm

### ISSUE THREAD

**Issue by metaclips** [dprint/dprint#725]
Using the `dprint.json` config file below and running  `dprint check` fails with a `No files found to format with the specified plugins at path. You may want to try using ``dprint output-file-paths`` to see which files it's finding.`
```json
{
    "includes": [
        "**/Cargo.toml"
    ],
    "plugins": [
        "https://plugins.dprint.dev/toml-0.5.4.wasm"
    ]
}
```

Running `dprint output-file-paths` prints our Cargo.toml files to be scanned.

**Comment by dsherret** [comment]
@metaclips what version of dprint are you using? I tried to reproduce this in the latest, but I can't. Also, in the latest dprint version you can remove the "includes" if you don't want to bother with it.

**Comment by metaclips** [comment]
Aha! I believe the issue is created when you call dprint with the `--config` arg, can you confirm?

**Comment by dsherret** [comment]
Possibly. Could you provide more details about the setup? Is the config file in a sub directory?

**Comment by dsherret** [comment]
This is fixed in 0.40.1. It was a regression in 0.40.0.

**Comment by metaclips** [comment]
Thanks, thanks, thanks.
